### DISCUSSION TITLE
3.4.1rc0

### DISCUSSION THREAD

**Discussion by snowman2** [pyproj4/pyproj#1198] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/pyproj4/pyproj/releases/tag/3.4.1rc0'>3.4.1rc0</a>.</em>

**Comment by snowman2** [comment]
The plan is to release with PROJ 9.1.0 in the wheels (see #1192).

Release Candidate Status:

- [x] Linux ARM wheels (https://github.com/pyproj4/pyproj-wheels/pull/55)
- [x] Conda Forge (https://github.com/conda-forge/pyproj-feedstock/pull/127)
- [x] Debian (@sebastic)
- [x] Fedora (@jdkloe)

Plan to release in a few days. Testing is always appreciated, but definitely not required.

**Comment by jdkloe** [comment]
Just tested the tar ball on Fedora rawhide and the build, install and test steps all passed. So this looks good.

Versions:
python3 --version
Python 3.11.1
gcc --version
gcc (GCC) 12.2.1 20221121 (Red Hat 12.2.1-4)
proj
Rel. 9.1.1, December 1st, 2022
Cython-0.29.32-1.fc37


Thanks for your efforts!

Jos

**Comment by sebastic** [comment]
Had to patch out use of `runtime_library_dirs` which doesn't make sense when `library_dirs` are in the `ld.so.conf` search paths. Lintian complains about the RPATH in the libraries otherwise.

The amd64 & arm64 builds succeeded (with PROJ 9.1.1):

 https://buildd.debian.org/status/package.php?p=python-pyproj&suite=sid

**Comment by snowman2** [comment]
Thanks @jdkloe @sebastic 👍
